Dockerfile

```diff
@@ -3,7 +3,7 @@
 FROM navikt/node-express:16 as builder-base
 USER root
 # Vi installerer compilere slik at node-pakken "sharp" kan bygge mot vips-dev image conversion biblioteket
-RUN apk --no-cache add curl binutils make gcc g++ --repository http://dl-cdn.alpinelinux.org/alpine/3.15/community/ vips-dev=8.12.2-r5
+RUN apk --no-cache add curl binutils make gcc g++ --repository http://dl-cdn.alpinelinux.org/alpine/3.15/community/ vips-dev=8.13.3-r1
```

Comment:
Hva er egentlig dette?

Reply:
Ett eller annet vi trenger pga. sharp pakken. Som igjen brukes for å håndtere heic/heif bilder 😐